### DISCUSSION TITLE
API fixes due to Gogoanime rebrand to Anitaku.to

### DISCUSSION THREAD

**Discussion by NYT92** [amvstrm/amvstrm#34] [Announcements]
Gogoanime will now switch to Anitaku.to the domain and we roll out the fixes as soon as possible. The most affected routes are ``/api/v1`` and ``/api/v2/stream`` because it relies on the gogoanime data.

We will set the website to maintenance mode from now on and stay tuned for updates.

**Comment by NYT92** [comment]
Sorry for the delay but the **ISSUE has been fixed**. I just found out about the error while I was browsing for anime and the response returned empty. I am sorry for that and I will implement a better error handler for the watch page too...

There might be a slow response due to website overload and our scraping still working as normal.

I will push the code after deploying my code to the server.

**Reply by avinave14102001** [reply]
> Sorry for the delay but the **ISSUE has been fixed**. I just found out about the error while I was browsing for anime and the response returned empty. I am sorry for that and I will implement a better error handler for the watch page too...
> 
> There might be a slow response due to website overload and our scraping still working as normal.
> 
> I will push the code after deploying my code to the server.

Anitaku.to as be not working yesterday or today 9 December 2024 this as be coming go to website error 522 as be check provide solutions

**Comment by avinave14102001** [comment]
Anitaku.to as be not working yesterday or today 9 December 2024 this as be coming go to website error 522 as be check provide solutions

**Reply by sandipansingh** [reply]
@avinave14102001 

https://api.animesage.online/

and for endpoint documentation go to https://docs.animesage.online/

Use this
